tools/SetupFlow/DevHome.SetupFlow/ViewModels/QuickstartPlaygroundViewModel.cs

```diff
@@ -604,6 +610,17 @@ public void OnQuickstartSelectionChanged()
             ConfigureForProviderSelection();
         }
     }
+
+    [RelayCommand]
+    private void OnLoaded()
+    {
+        var isEnabled = _experimentationService.IsFeatureEnabled("QuickstartPlayground");

```

Comment:
Any easy way we can use / share the feature name const string in the other file (QuickstartPlaygroundFlowFeatureName).

Reply:
Yup! We can use the const string directly instead of a hardcoded string.